package/MDAnalysis/analysis/hole.py

```diff
@@ -1283,6 +1297,10 @@ def run(self, **kwargs):
         protein = self.universe.select_atoms(self.selection)
         for q, ts in zip(self.orderparameters[start:stop:step], self.universe.trajectory[start:stop:step]):
             logger.info("HOLE analysis frame %4d (orderparameter %g)", ts.frame, q)
+            if self._dynamic_cpoint:
+                hole_kw['cpoint'] = self.cpoint.center_of_geometry()
+                logger.info("New dynamic CPOINT %r for frame %r", hole_kw['cpoint'], ts.frame)
```

Comment:
This has the potential to be super verbose. Wouldn't it be more convenient to save the cpoint in an array instead?

Reply:
By verbose you mean "giving output every frame"? Yes, I'll have to check what else is printed. In practice, I would not run this on 1000s of frames because it is pretty slow. 

> Wouldn't it be more convenient to save the cpoint in an array instead?

You mean as an attribute of the class for further investigation later? That's a good idea.

Reply:
For right now, I changed all per-frame logging to `logger.debug`. Should use progressmeter for some...